x-pack/plugins/cloud_security_posture/public/common/constants.ts

```diff
@@ -89,11 +89,7 @@ export const cloudPostureIntegrations: CloudPostureIntegrations = {
         benchmark: i18n.translate('xpack.csp.cspmIntegration.gcpOption.benchmarkTitle', {
           defaultMessage: 'CIS GCP',
         }),
-        disabled: true,
```

Comment:
disable if integration is not above 1.5

OR

allow to chose it but render a different UI asking to use the integration version above 1.5